src/env.ts

```diff
@@ -0,0 +1,9 @@
+import * as envalid from 'envalid'
+import dotenv from 'dotenv'
+
+dotenv.config()
+
+export default envalid.cleanEnv(process.env, {
```

Comment:
Is adding the `envalid` lib here the best way to set a default value for these env vars? On the validation aspect, all env vars should be strings already

Reply:
I've removed `envalid` and `dotenv`, using `process.env` and default values if the env is missing instead

Reply:
One consideration that might have been overlooked here is consistency. Whilst I agree we don't need `envalid` to parse these strings we might want to load them using `.env` files at some point. We do this the same way everywhere else so is there much to gain/lose by continueing to follow this pattern?

Another thought is whether this should be configured by the starup args like the other service arguments?

Reply:
My point is this is fairly inconsistent with both the way this service is configured and the way we ocnfigure elsewhere. I see two options that would be consistent with something:

1. If the plan is to configure this service like we do elsewhere (remove the cli args and move to envs / env files) then I'd introduce envalid now as we'll use that for validating the other args. 
2. If the plan is to continue to configure via cli args do that

Reply:
Moving from cli args to envs seems like a larger question. I've changed this to use cli args so it matches convention/minimal changes. We can see if env files are something we want to use in the future